arwen/helpers.go

```diff
@@ -86,3 +87,8 @@ func GetSCCode(fileName string) []byte {
 
 	return code
 }
+
+func TimeTrack(start time.Time, message string) {
+	elapsed := time.Since(start)
+	fmt.Println(message, "duration", elapsed)
```

Comment:
Do we have access to a logger here, for `logger.Trace` instead of `fmt.Println`?

Reply:
No, we don't at this moment. Only when Arwen will reference the "true" logger.